### PR TITLE
chore(manypets): add logs for signing flow

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2715** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2715" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2715?utm_source=stack-comment).

File: apps/store/src/features/manyPets/ManyPetsMigrationPage/ManyPetsMigrationPage.tsx [hunk 1]
```diff
@@ -242,26 +243,35 @@ const useSignMigration = (
 
   const handleSubmitSign: FormEventHandler<HTMLFormElement> = useCallback(
     async (event) => {
-      if (!shopSession) return
-
       event.preventDefault()
 
+      if (!shopSession) return
```
Comment (robinandeer) [inline apps/store/src/features/manyPets/ManyPetsMigrationPage/ManyPetsMigrationPage.tsx:248]: I guess there's no harm in re-ordering these...

**Review by robinandeer — COMMENTED** [review]
NICE!

**Review by robinandeer — APPROVED** [review]
NICE!

File: apps/store/src/features/manyPets/ManyPetsMigrationPage/ManyPetsMigrationPage.tsx (hunk 1 above)
Comment (guilhermespopolin) [reply apps/store/src/features/manyPets/ManyPetsMigrationPage/ManyPetsMigrationPage.tsx:248]: It shouldn't harm :)

**Pull request merged**